inlineplz/linters/htmlhint.py

```diff
@@ -3,10 +3,29 @@
 from __future__ import unicode_literals
 
 import dirtyjson as json
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "import dirtyjson as json" (wrong-import-order)`